src/interface/src/styleguide/paginator/paginator.component.scss

```diff
@@ -0,0 +1,234 @@
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 14px;
+  text-align: center;
+  cursor: pointer;
+}
+
+.page-button:hover {
+  background-color: $color-light-gray;
+  color: $color-black;
+}
+
+.page-button:active {
+  background-color: $color-light-gray;
+  color: $color-black;
+}
+
+.selected-button {
+  background-color: $color-standard-blue;
+  color: white;
+}
+
+.selected-button:active {
+  background-color: $color-standard-blue;
+  color: white;
+}
+
+.selected-button:hover {
+  background-color: $color-standard-blue;
+  color: white;
+}
+
+.spacer-button {
+  color: $color-md-gray;
+  background-color: white;
+  border: none;
+  padding: 0px;
+  height: 38px;
+  width: 30px;
+  font-size: 14px;
+  border-left: 1px solid $color-soft-gray;
+}
+
+.arrow-icon {
+  height: 20px;
+  width: 20px;
+  font-size: 20px;
+  line-height: 26px;
+}
+
+.ellipse-icon {
+  height: 20px;
+  width: 20px;
+  font-size: 20px;
+  line-height: 36px;
+}
+
+.first-page-button {
+  border-top-left-radius: 10%;
+  border-bottom-left-radius: 10%;
+  padding-left: 12px;
+  .mat-icon {
+    font-size: 14px;
+    height: 100%;
+    width: 14px;
+    line-height: 22px;
+  }
+}
+
+.last-page-button {
+  border-top-right-radius: 10%;
+  border-bottom-right-radius: 10%;
+  border-right: 1px solid $color-soft-gray;
+  .mat-icon {
+    font-size: 14px;
+    height: 100%;
+    width: 14px;
+    line-height: 22px;
+  }
+}
+
+.page-button:disabled {
+  background-color: $color-white;
+  color: $color-text-gray;
+}
+
+.pages-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 100%;
+}
+
+.nav-control-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.nav-controls {
+  display: flex;
+  flex-direction: row;
+}
+
+.page-select-container {
+  width: 200px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: 8px;
+  display: none;
+}
+
+.nav-label {
+  @include small-input-label();
+  margin: 0px;
+  width: 180px;
+  text-align: right;
+  color: $color-black;
+}
+
+.per-page-picker {
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  align-items: center;
+  width: 220px;
+  justify-content: space-between;
+  gap: 8px;
+}
+
+.picker-label {
+  @include small-input-label();
+  margin: 0px;
+  width: 180px;
+  text-align: right;
+  color: $color-black;
+}
+
+.picker-field {
+  width: 100px;
+  padding-top: 0px;
+  border: $color-black;
+
+  &.disabled {
+    opacity: 0.6;
+  }
+}
+
+::ng-deep .per-page-item.mat-mdc-option.mdc-list-item {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 14px;
+  text-align: center;
+  color: $color-md-gray;
+}
+::ng-deep .nav-page-item.mat-mdc-option.mdc-list-item {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 14px;
+  text-align: center;
+  color: $color-md-gray;
+}
+
+::ng-deep .picker-field .mdc-notched-outline .mdc-notched-outline__trailing {
+  border-color: $color-soft-gray !important;
```

Comment:
Try out redefining this variables instead of using !important

<img width="710" alt="Screen Shot 2024-06-18 at 13 28 35" src="https://github.com/OurPlanscape/Planscape/assets/358892/1125b04a-ebc0-41f7-b053-b780fa393b3c">

Additionally, you might be able to avoid going deep into the component markup and just apply your own class that overrides the css variable.